<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="content">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblLogin">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Usu√°rio</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsuario">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>169</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="inputMask">
      <string notr="true">99999999999</string>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
     <property name="placeholderText">
      <string>Digite seu CPF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSenha">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Digite a senha</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxEsqueciSenha">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Esqueci a senha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:#A9A9A9;
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #C0C0C0;
	border: 2px solid #ffffff;

}
QPushButton:pressed{
	background-color:rgb(250,230,0);
	border: 2px solid rgb(255,165,24);
	color: rgb(35,35,35);
}</string>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
